### DISCUSSION TITLE
Should we release 0.69.12?

### DISCUSSION THREAD

**Discussion by kelset** [reactwg/react-native-releases#72] [Patches]
Released. The 0.69.12 is the latest patch for 0.69 the release crew will prepare as, with the release of 0.72.0, version 0.69 exited the support window. 
---
Current Release: [0.69.11](https://github.com/facebook/react-native/releases/tag/v0.69.11)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1. 

#### Local commits to backport to main

1.

**Comment by kelset** [comment]
from @robhogan:
> Nominate this one for the RN-side fix to https://github.com/facebook/react-native/issues/36794 (to accompany a bump to Metro 0.70.4).
> - https://github.com/facebook/react-native/pull/37501 (https://github.com/facebook/react-native/commit/671ea383fe45dd9834a0c0481360de050df7f0c9)

**Reply by kelset** [reply]
bump to do:
* Metro 0.70.4
* CLI 8.0.7 https://github.com/react-native-community/cli/commits/8.x

**Reply by cipolleschi** [reply]
picked these two.

**Comment by mcolotto** [comment]
Hello, is there any chance we could get https://github.com/facebook/react-native/commit/c05d822f7daa92e8af2ec2cd97a9897425624cc2 into 0.69?

**Reply by cipolleschi** [reply]
picked this commit.

**Comment by pooja-hcom** [comment]
@kelset when can we expect 0.69.12 to go live?

**Reply by cortinico** [reply]
Please don't ask ETA question as a lot of this work is _best effort_. We'll release 0.69.12 whenever it's ready. If you're waiting for a specific fix, you should consider updating to the latest major which most likely include that fix already.

**Comment by kelset** [comment]
headsup: with the release of 0.72.0, this minor (0.69) now moves into unsupported status, so don't expect further releases to be done. Check out [the release policy](https://github.com/reactwg/react-native-releases#releases-support-policy) for more details.
